sacn/messages/root_layer_test.py

```diff
@@ -43,3 +44,12 @@ def test_root_layer_bytes():
     # cid
     shouldBe.extend(cid)
     assert packet.getBytes() == shouldBe
+
+
+def test_int_byte_transitions():
+    failedNums = []
+    for input_i in range(64000):
+        converted_i = byte_tuple_to_int(tuple(int_to_bytes(input_i)))
+        if input_i != converted_i:
+            failedNums.append(f'{input_i} != {converted_i}')
+    assert [] == failedNums
```

Comment:
I'd argue it would be easier to just assert the correct value right away:
```python
for input_i in range(64000):
        converted_i = byte_tuple_to_int(tuple(int_to_bytes(input_i)))
        assert input_i == converted_i
```

Reply:
done.  I'd originally written the other way while trying to sort where universe number conversions were breaking -- so a summary had more debugging value at the time.  As a unit test, this way is simpler.